internal/policy/policy.go

```diff
@@ -168,7 +168,6 @@ func WithStoreIdentifier(p *policyv1.Policy, storeIdentifier string) *policyv1.P
 		p.Metadata = &policyv1.Metadata{}
 	}
 
-	//nolint:staticcheck
```

Comment:
Unfortunately the linter is [flaky](https://github.com/golangci/golangci-lint/issues/3228) - this nolint is actually correct and still needed.

Reply:
Ah my bad, that's me running `make lint`. Restored.